googletest/test/gtest_unittest.cc

```diff
@@ -4284,25 +4284,29 @@ TEST(SuccessfulAssertionTest, SUCCEED) {
 // Tests that Google Test doesn't track successful EXPECT_*.
 TEST(SuccessfulAssertionTest, EXPECT) {
   EXPECT_TRUE(true);
-  EXPECT_EQ(0, GetUnitTestImpl()->current_test_result()->total_part_count());
+  EXPECT_EQ(1, GetUnitTestImpl()->current_test_result()->total_part_count());
```

Comment:
We need to test both flag settings.

Reply:
The flag will not have an influence on the stored information, only on the output itself, see https://github.com/google/googletest/pull/3678/files/9c2d5c84300a271a4cc503ee606120ce57b52453?authenticity_token=GA9v%2FZzi5u38QObqbF5CCdesUinLL%2B3vls9ZNOti9nV7Osp479Bdved2P357pvWozUiRI8F%2B8zd30q2PjY2dog%3D%3D&file-filters%5B%5D=.cc#diff-bdb2e6c83624322651521250ab8b6249c086965371a73c75ed19f40e82d05b53R3787